gwvolman/tasks.py

```diff
@@ -369,10 +369,14 @@ def build_tale_image(task, tale_id, force=False):
 
     logging.info('Last build time {}'.format(last_build_time))
 
+    image_changed = tale["imageId"] != tale["imageInfo"].get("imageId", tale["imageId"])
```

Comment:
We'll need a migration step to add the imageId to imageInfo for existing tales, otherwise I won't this will always be false? Another option is to also force rebuild if there's no imageId on imageInfo.

Reply:
I'll change that to simply `tale["imageInfo"].get("imageId")` which will force build.